spec/helper/get_docker_params_changed.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+def get_docker_params_changed(opts) # rubocop:disable Metrics/CyclomaticComplexity,Metrics/PerceivedComplexity
+  require 'open3'
+  require 'json'
+  return_value = 'NO_CHANGE_DETECTED'
```

Comment:
Would perhaps suggest
```suggestion
  return_value = 'No changes detected'
```
...to keep it consistent with the rest of the output